spyder/plugins/application/container.py

```diff
@@ -284,73 +279,105 @@ def _check_updates_ready(self):
 
         # Adjust the checkbox depending on the stored configuration
         option = 'check_updates_on_startup'
-        check_updates = self.get_conf(option)
-        box.set_checked(check_updates)
+        box.set_checked(self.get_conf(option))
+
+        header = _(
+            "<b>Spyder {} is available!</b> "
+            "<i>(you&nbsp;have&nbsp;{})</i><br><br>"
+        ).format(latest_release, __version__)
 
         if error_msg is not None:
-            msg = error_msg
-            box.setText(msg)
+            box.setText(error_msg)
             box.set_check_visible(False)
             box.exec_()
-            check_updates = box.is_checked()
-        else:
-            if update_available:
-                if self.application_update_status:
-                    self.application_update_status.set_status_pending(
-                        latest_release=latest_release)
 
-                header = _("<b>Spyder {} is available!</b> "
-                           "<i>(you have {})</i><br><br>").format(
-                    latest_release, __version__)
-                content = ""
-                footer = _(
+        elif update_available:
+            # Update using our installers
+            if (
+                not sys.platform.startswith("linux")
+                or parse(latest_release) >= parse("6.0.0")
+            ):
+                box.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+                box.setDefaultButton(QMessageBox.Yes)
+
+                msg = (
+                    header +
+                    _("Would you like to automatically download "
+                      "and install it? <br><br>")
+                )
+
+                if not is_pynsist() and not running_in_mac_app():
+                    msg += _(
+                        "Clicking <b>Yes</b> will download and install the "
+                        "latest version using Spyder's installer, leaving "
+                        "your existing installation untouched.<br><br>"
+                    )
```

Comment:
I like your recommendation, but I would make an a slight modification.
```suggestion
                if not is_pynsist() and not running_in_mac_app():
                    installers_url = (
                        "http://docs.spyder-ide.org/current/installation.html"
                        "#standalone-installers"
                    )
                    msg = (
                        header +
                        _("Would you like to automatically download and "
                          "install it using Spyder's installer?"
                          "<br><br>"
                          "We recommend Spyder's installer because it is "
                          "<a href=\"{}\">more stable</a>. "
                          "Note that this will leave your existing "
                          "installation untouched.").format(installers_url)
                    )
                else:
                    msg = (
                        header +
                        _("Would you like to automatically download "
                          "and install it? <br><br>")
                    )```

Reply:
I agree with @CAM-Gerlach's suggestion.

Reply:
LGTM except that there is a space missing after "your existing installation" (line 333).

Reply:
@jitseniesen we also caught that and it should be fixed in the latest commit.